TESTING.md

```diff
@@ -292,10 +292,8 @@ export const MyVisualStory = () => {
 
 MyVisualStory.story = {
   parameters: {
-    chromatic: {
-      disabled: false,
-    },
-  }
+    ...enableSnapshots(),
+  },
```

Comment:
An alternate would be a CSF-only helper:

```ts
enableSnapshots(MyVisualStory)
```

That seems a little more magical, but I'm looking at a helper function to hide a story and currently that cannot be done with parameters:

```ts
MyVisualStory.story = {
  ...testingOnly(),
  parameters: {
    ...enableSnapshots(),
  }
}
```

This is less "magic", but doesn't seem easier to understand. Alternatively, the "decorator"-type pattern seems like the helper functions are at the same level of abstraction:

```ts
testingOnly(MyVisualStory)
enableSnapshots(MyVisualStory)

// or
testingOnly(enableSnapshots(MyVisualStory))
```